src/screens/account/account.tsx

```diff
@@ -14,19 +15,30 @@ import { Icon } from '../../components/icon';
 import { AccountSettings } from './components/account-settings/account-settings';
 import { withNavigationParams, INavigationProps } from '../../navigation/with-navigation-params';
 import { AccountAddress } from '../../components/account-address/account-address';
+import { formatAddress } from '../../core/utils/format-address';
+import { BLOCKCHAIN_INFO } from '../../core/blockchain/blockchain-factory';
 
 export interface IProps {
     navigation: NavigationScreenProp<NavigationState, NavigationParams>;
     styles: ReturnType<typeof stylesProvider>;
 }
 
 export interface IReduxProps {
-    accountx: IAccountState;
+    account: IAccountState;
+    transactions: { [id: string]: ITransactionState };
 }
 
+export const mapStateToProps = (state: IReduxState, ownProps: INavigationParams) => {
+    return {
+        account: state.wallets[state.app.currentWalletIndex].accounts[ownProps.accountIndex],
```

Comment:
this will change after #156 will be merged, `walletIndex` is changed to `walletId`